ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -224,29 +224,9 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
 
     kmLog("Setting language: \(kb.languageID)_\(kb.id)", checkDebugPrinting: true)
 
-    // FIXME: kb.version is not respected. Ideally we should be able to trust that the version number in UserDefaults
-    // is-to-date but it is sometimes not updated.
-    guard let kbVersion = latestKeyboardFileVersion(withID: kb.id) else {
-      kmLog("Could not set keyboardID to \(kb.id) because the keyboard file does not exist",
```

Comment:
How does this affect #419?

Reply:
Should fix it but I'd haven't tested. #470 makes it easier to verify that there is a keyboard file whose version matches what the app thinks the version is.